src/scaict_uwu/service_container.py

```
@@ -0,0 +1,91 @@
+import importlib
+import sys
+
+
+class ServiceContainer:
+    """
+    Service locator for SCAICT-uwu core services.
+
+    Refer to service_wiring.py for the default implementations.
+    """
+
+    _services: dict = {}
```

Ruff (0.8.0)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)